src/domains/plugin/components/PluginHeader/components/PluginSpecs/PluginSpecs.tsx

```diff
@@ -79,7 +79,10 @@ export const PluginSpecs = ({ author, category, url, version, isOfficial, size }
 							<a
 								data-testid="PluginSpecs__url"
 								href="/"
-								onClick={() => openExternal(url!)}
+								onClick={(evt) => {
+									evt.preventDefault();
+									openExternal(url!);
+								}}
```

Comment:
can't we use `<Link to={url!} isExternal>` here?

Reply:
oh looks better! pushed 👍🏻